osc-ui/src/main/java/org/osc/core/broker/view/maintenance/PluginsLayout.java

```diff
@@ -100,15 +101,16 @@ public PluginsLayout(BundleContext ctx, ImportPluginServiceApi importPluginServi
         this.plugins.addContainerProperty(PROP_PLUGIN_INFO, String.class, null);
         this.plugins.addContainerProperty(PROP_PLUGIN_DELETE, Button.class, null);
 
+        this.plugins.setColumnWidth(PROP_PLUGIN_INFO, PLUGIN_INFO_COLUMN_WIDTH);
```

Comment:
can we add a tooltip so we can look at the longer error? or if we can expand the column and scroll, that should be fine as well.